packages/cli/src/commands/start.ts

```diff
@@ -123,6 +128,20 @@ export async function startAgent(
 
   const characterPlugins: Plugin[] = [];
 
+  // Ensure plugin-sql is included
+  if (!encryptedChar.plugins.includes('@elizaos/plugin-sql')) {
```

Comment:
not really sure about this since user could create their own custom plugin for database adapter 🤔

Reply:
Agreed that we shouldn’t hardcore and instead find another way to handle defaults